src/EnergyPlus/MixedAir.cc

```diff
@@ -4681,6 +4681,15 @@ void CalcOAMixer(EnergyPlusData &state, int const OAMixerNum)
     // Mixed air temperature is calculated from the mixed air enthalpy and humidity ratio.
     state.dataMixedAir->OAMixer(OAMixerNum).MixTemp =
         Psychrometrics::PsyTdbFnHW(state.dataMixedAir->OAMixer(OAMixerNum).MixEnthalpy, state.dataMixedAir->OAMixer(OAMixerNum).MixHumRat);
+
+    // Check for saturation temperature > dry-bulb temperature and modify temperature at constant enthalpy
+    auto T_sat =
```

Comment:
Don't use auto for scalars. And use definition when possible. Here it's Real64 T_sat. This looks correct otherwise. This makes me wonder how many other mixing calcs need this check. Things like setCoolCoilInletTempForZoneEqSizing (and HumRat) in sizing routines, HXs, etc.

    Real64 PsyTsatFnHPb(